.travis.yml

```diff
@@ -1,6 +1,17 @@
 language: python
 python:
   - "2.7"
+env:
```

Comment:
I have a `before_script` entry calling `tox -e py`. It should run before travis does the itests. This will allow users to catch syntax errors without having to wait for the long tests to run.